storage-model/src/main/java/no/unit/nva/publication/storage/model/StorageModelConstants.java

```diff
@@ -0,0 +1,75 @@
+package no.unit.nva.publication.storage.model;
+
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class StorageModelConstants {
+
+    public static final String HOST_ENV_VARIABLE_NAME = "HOST";
+    public static final String NETWORK_SCHEME_ENV_VARIABLE_NAME = "SCHEME";
+    public static final String MESSAGES_PATH_ENV_VARIABLE_NAME = "MESSAGES_PATH";
+    public static final String URI_EMPTY_FRAGMENT = null;
+    public static final String PATH_SEPARATOR = "/";
+    public static final String MISSING_ENV_VARIABLE_MESSAGE =
+        StorageModelConstants.class.getName() + "is missing an env variable";
+    private static final Logger logger = LoggerFactory.getLogger(StorageModelConstants.class);
+    private static final String DEFAULT_SCHEME = "https";
+    private static final String ROOT = PATH_SEPARATOR;
+    private static final String DEFAULT_MESSAGES_PATH = "/messages";
+
+    private static StorageModelConstants INSTANCE = defaultInstance();
+
+    public final String host;
+    public final String scheme;
+    public final String messagePath;
+
+    public StorageModelConstants(Environment environment) {
```

Comment:
can this be set to private since you access the class through getInstance() and only want one?